sirepo/template/madx.py

```diff
@@ -721,13 +721,16 @@ def _extract_report_twissFromParticlesAnimation(data, run_dir, filename):
         del res[f'alpha_{dim}']
         res[f'bet{dim}'] = res[f'beta_{dim}']
         del res[f'beta_{dim}']
-    assert set(res.keys()) == set(_PTC_OBSERVE_TWISS_COLS), \
-        f'unknown ptc twiss columns={set(res.keys())} expected={_PTC_OBSERVE_TWISS_COLS}'
+    assert set(res.keys()) == set(PTC_OBSERVE_TWISS_COLS), \
+        f'unknown ptc twiss columns={set(res.keys())} expected={PTC_OBSERVE_TWISS_COLS}'
     return extract_parameter_report(
         data,
         results=PKDict(res),
     )
 
+def _extract_report_twissFromParticlesAnimation(data, run_dir, filename):
```

Comment:
I think this can be removed.

Reply:
the private method is needed for madx since madx visualization does code generation